### PR TITLE
Seeding data augmentation

### PR DESCRIPTION
**Pull request by forklady42**
I did some looking into how the numpy seeding worked for data augmentation.

Initially, I checked with r and rint were for workers across epoches.

epoch_1_worker_0 | epoch_1_worker_1 | epoch_2_worker_0 | epoch_2_worker_1 | epoch_3_worker_0 | epoch_3_worker_1
-- | -- | -- | -- | -- | -- 
0.4388784397520520, 0 | 0.4388784397520520, 0 | 0.4388784397520520, 0 | 0.4388784397520520, 0 | 0.4388784397520520, 0 | 0.4388784397520520, 0
0.8585979199113830, 2 | 0.8585979199113830, 2 | 0.8585979199113830, 2 | 0.8585979199113830, 2 | 0.8585979199113830, 2 | 0.8585979199113830, 2
0.09417734788764950, 0 | 0.09417734788764950, 0 | 0.09417734788764950, 0 | 0.09417734788764950, 0 | 0.09417734788764950, 0 | 0.09417734788764950, 0
0.9756223516367560, 2 | 0.9756223516367560, 2 | 0.9756223516367560, 2 | 0.9756223516367560, 2 | 0.9756223516367560, 2 | 0.9756223516367560, 2
0.7860643052769540, 2 | 0.7860643052769540, 2 | 0.7860643052769540, 2 | 0.7860643052769540, 2 | 0.7860643052769540, 2 | 0.7860643052769540, 2
0.12811363267554600, 2 | 0.12811363267554600, 2 | 0.12811363267554600, 2 | 0.12811363267554600, 2 | 0.12811363267554600, 2 | 0.12811363267554600, 2
… | … | … | … | … | … 

As shown here, the same random numbers are generated in the same order across all workers and repeated across epochs. This reduces the actual randomness used in data augmentation.

**This is not an immediate issue because data is shuffled between epochs.** It is unlikely that the same piece of data will be run in the same place in order and thus use the same generated random value. However, it is misleading if we are expecting the augmentation code to be fully randomized. We should be clear about our expectations.

If we want true randoms, the simplest fix to use pytorch's built in RNG. I also changed `random_state` to `random_seed` to match the usual language in pytorch and numpy. 

Updated r and rint values with the changes in this PR:

epoch_1_worker_0 | epoch_1_worker_1 | epoch_2_worker_0 | epoch_2_worker_1 | epoch_3_worker_0 | epoch_3_worker_1
-- | -- | -- | -- | -- | -- 
0.28845077753067000, 0 | 0.992135226726532, 1 | 0.12274551391601600, 0 | 0.6259416937828060, 1 | 0.5628141760826110, 1 | 0.9500988721847530, 1
0.5059208273887630, 2 | 0.7314887642860410, 1 | 0.4147964119911190, 2 | 0.28725552558898900, 0 | 0.1841621994972230, 2 | 0.3092734217643740, 1
0.7570074200630190, 0 | 0.971265435218811, 1 | 0.21043217182159400, 1 | 0.01940363645553590, 2 | 0.2232559323310850, 2 | 0.5673806071281430, 1
0.6470523476600650, 0 | 0.021842360496521000, 1 | 0.8638843894004820, 1 | 0.14560383558273300, 1 | 0.39373016357421900, 1 | 0.22419828176498400, 1
0.44518280029296900, 2 | 0.024007022380828900, 0 | 0.7812286019325260, 1 | 0.6654463410377500, 1 | 0.9641874432563780, 1 | 0.1305629014968870, 1
0.26160913705825800, 0 | 0.8439620733261110, 1 | 0.12506234645843500, 2 | 0.8457986116409300, 2 | 0.6085056066513060, 0 | 0.4428870677948000, 0
… | … | … | … | … | …

### CODE REVIEW

File: src/electrai/configs/config.yaml
Comment (hanaol) [inline src/electrai/configs/config.yaml]: This file is no longer needed. Could you please remove it as part of this PR?

**Review by hanaol — COMMENTED** [review]
Left a minor comment.

File: src/electrai/configs/config.yaml
Comment (hanaol) [reply src/electrai/configs/config.yaml]: I agree that my comment is unrelated to this PR and should be discussed later. We could later convert it to a general configuration file, along with instructions on how to handle different datasets and dataloaders.

I'll go ahead and merge the PR.
